src/components/AdvertBanner/styles.ts

```diff
@@ -0,0 +1,12 @@
+import { View } from 'native-base'
+import { css, styled } from 'styled-components'
+
+export const AdvertWrapper = styled(View)<{ isHidden: boolean }>(
+  ({ isHidden }) => css`
+    align-items: center;
+    width: 100%;
+    max-height: 106px;
+    height: ${isHidden ? '106px' : 'auto'};
```

Comment:
why we need this condition?

Reply:
Because we need to have 100% width and adaptive height for right proportion of advert is provided and if advertising is not provided, then the block should take up space